scripts/docker/dev/start-django.sh

```diff
@@ -21,6 +21,13 @@ fi
 npm run devw&
 popd
 
+pushd akvo/rsr/spa
+if [[ ! -d "node_modules" ]]; then
+  npm install
```

Comment:
Should this be a `yarn install`? I see a `yarn.lock` file was committed to the repo. I'm guessing we want to do a `yarn install` ? @martinchristov ?

Reply:
I'm a little reluctant to add yarn to the image because it will add more time for the image build process, using `npm install` here everything can run well and no problems found.

I can update it and use `yarn` if it is a must.

Reply:
If `yarn` is the new standard, we can switch to using `yarn` for both the spa and other static assets. But, having a `yarn.lock` file, and not using `yarn` seems like a smell, to me.

Reply:
I think `yarn` vs. `npm` is only a matter of preference, but I could be wrong.

I understand your concern. We can do one of the following;
- remove `akvo/rsr/spa/package-lock.json`, add `yarn` binary to the images, and use yarn to manage the SPA dependencies.
- remove `akvo/rsr/spa/yarn.lock`, and continue using `npm` to manage the SPA dependencies.

Please give your suggestion and I will update the PR.

Reply:
Up to @martinchristov to decide. I personally do not know the difference between them. Probably if we wait for a week, there will be a new "standard" 😆

Reply:
Hey guys. We could drop yarn and just stick to plain npm. I've been using yarn because it used to be faster than npm, but I just checked the latest state of affairs and npm has caught up to speed.

Reply:
npm it is then 😃

Reply:
I wrote the above after doing a quick research, but later when I was running tests of how `npm install` (latest version of npm) is performing I discovered that it still takes an unreasonably large amount of time. Even when there is nothing new to install npm takes some 15-20 seconds just to figure this out, whereas yarn takes just 1 second.
Perhaps we should use yarn after all.

Reply:
We could use `npm` for now, to get this PR merged quicker. And then later move all the `npm` stuff to `yarn`.